app/scripts/controllers/overview.js

```diff
@@ -440,7 +440,7 @@ angular.module('overview', ['stats', 'checks', 'ngSanitize', 'events', 'filters_
       return 'SELECT CURRENT_TIMESTAMP AS last_timestamp, ' +
         '(ended / 10000) * 10000 + 5000 AS ended_time, ' +
```

Comment:
What about this spot?
```sql
(ended / 10000) * 10000 + 5000 AS ended_time
```

Reply:
That is just a timestamp, all good.

That probably could be changed to use `date_bin` ...
```sql
date_bin('10 seconds'::INTERVAL, ended, 0) + 5000 AS ended_time
```

... but not really necessary

Reply:
Thanks for the suggestion. Feel free to submit another patch, or let me know if I should do it. If you think it's not required _now_, let's keep this very patch minimal.